Sources/PostgreSQL/Database/PostgreSQLTransportConfig.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+import NIOOpenSSL
+
+
+public struct PostgreSQLTransportConfig {
+    /// Does not attempt to enable TLS (this is the default)
+    public static var cleartext: PostgreSQLTransportConfig {
```

Comment:
This should be `clearText` (with a capital T) so it matches Vapor's usage of `plainText`.